### PR TITLE
Extend Dockerfile with additional labels

### PR DESCRIPTION
**Pull request by bastelfreak**
This is based on https://github.com/puppetlabs/bolt-vanagon/blob/main/docker/puppet-bolt/Dockerfile. The ARG stuff from the bolt Dockerfile might be a nice way to build for Puppet 7 and 8?

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,13 @@
 FROM ubuntu:22.04
 
-LABEL org.label-schema.maintainer="betadots GmbH <info@betadots.de>"
+LABEL org.label-schema.maintainer="betadots GmbH <info@betadots.de>" \
+      org.label-schema.vendor="betadots" \
+      org.label-schema.url="https://github.com/betadots/pdc" \
+      org.label-schema.name="Puppet Development Container" \
+      org.label-schema.license="Apache-2.0" \
```
Comment (rwaffen) [inline Dockerfile:7]: we dont have a licence yet, and dont we take agpl like in hdm?

**Comment by rwaffen** [comment]
was also looking for the arg stuff. might be a thing, looks easy enough atm

File: Dockerfile (hunk 1 above)
Comment (bastelfreak) [reply Dockerfile]: I had to do a bit of research about the scope first. This label is only, to my understanding, about our own software in the repo. Which is basically only the Dockerfile/Gemfile. I updated it to AGPL3 as well.

**Comment by rwaffen** [comment]
will update this pr after this is merged:
https://github.com/voxpupuli/gha-build-and-publish-a-container/pull/13

**Comment by bastelfreak** [comment]
@rwaffen should we merge this as is and do a release?

**Comment by rwaffen** [comment]
we can merge this, but need to work on the upstream gha to allow us to set the tag manually. atm tags are set automatically when you push to main (development) or when you push a tag (v1.2.3)

**Pull request merged**